wd_helper.au3

```diff
@@ -832,7 +829,7 @@ EndFunc   ;==>_WD_FrameList
 ;                  $bIsHidden           - information about visibility of frame - taken by WebDriver
 ; Return values .: Success - string
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
-;                  - $_WD_ERROR_Exception
+;                  - $_WD_ERROR_?????????
```

Comment:
need to be rethinked/checked

Reply:
Please take a look: [I hope the final version](https://github.com/Danp2/au3WebDriver/pull/434/commits/8cac3267a123d1fd4e5f99d88332a054a8e074b6)

Reply:
That's a **really** long list of functions. Is it really necessary to list each function that was used by `__WD_FrameList_Internal`? What other alternatives did you consider?

Reply:
See my review below.

Reply:
Asap

Reply:
Cool